src/solver/simulation/simulation.h

```diff
@@ -55,7 +56,8 @@ void SIM_RenseignementProblemeHebdo(const Antares::Data::Study& study,
                                     PROBLEME_HEBDO& problem,
                                     uint weekInTheYear,
                                     uint numSpace,
-                                    const int);
+                                    const int,
+                                    VAL_GEN_PAR_PAYS&);
```

Comment:
At this stage, we don't modify anymore the generated values: it should be const here and everywhere "below".